src/test/rpc/AmendmentBlocked_test.cpp

```diff
@@ -0,0 +1,84 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2017 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <test/jtx.h>
+#include <ripple/protocol/JsonFields.h>
+#include <ripple/app/misc/NetworkOPs.h>
+
+namespace ripple {
+
+class AmendmentBlocked_test : public beast::unit_test::suite
+{
+    void testBlockedMethods()
+    {
+        using namespace test::jtx;
+        Env env {*this};
+        auto const alice = Account {"alice"};
+        auto const gw = Account {"gateway"};
+        env.fund (XRP(10000), alice, gw);
+        auto const USD = gw["USD"];
+        auto const CNY = gw["CNY"];
+        env (trust (alice, USD(1000)));
+        env (trust (alice, CNY(1000)));
+        env (offer (alice, USD(1), XRP(1000)));
+        env.close();
+
+        auto current = env.current ();
+        auto jr = env.rpc ("ledger_accept") [jss::result];
+        BEAST_EXPECT (jr[jss::ledger_current_index] == current->seq ()+1);
+
+        jr = env.rpc ("ledger_current") [jss::result];
+        BEAST_EXPECT (jr[jss::ledger_current_index] == current->seq ()+1);
+
+        jr = env.rpc ("owner_info", alice.human()) [jss::result];
+        BEAST_EXPECT (jr.isMember(jss::accepted) && jr.isMember(jss::current));
+
+        // not tested (TODO):  path_find, submit, submit_multisigned
```

Comment:
I'd like to see tests for `submit`/`submit_multisigned` since the [docs](https://ripple.com/build/amendments/#amendment-blocked) explicitly mention `Cannot submit or process transactions`